tests/scipy_stats_test.py

```diff
@@ -49,6 +49,41 @@ def genNamedParametersNArgs(n):
 class LaxBackedScipyStatsTests(jtu.JaxTestCase):
   """Tests for LAX-backed scipy.stats implementations"""
 
+  @genNamedParametersNArgs(2)
+  def testBernoulliCdf(self, shapes, dtypes):
+    rng = jtu.rand_default(self.rng())
+    scipy_fun = osp_stats.bernoulli.cdf
+    lax_fun = lsp_stats.bernoulli.cdf
+
+    def args_maker():
+      x, logit = map(rng, shapes, dtypes)
+      x_dtype, p_dtype = x.dtype, logit.dtype
+      x = np.floor(x).astype(x_dtype)
```

Comment:
I would use `jtu.rand_int` for `x` here rathern than `rand_default` followed by `floor`.